docs/public/deploying-a-devfile-using-odo.adoc

```diff
@@ -91,42 +93,36 @@ In our example, we will be using `java-spring-boot` to deploy a sample https://s
 
 == Deploying a Java Spring Boot® component to an OpenShift cluster
 
-In this example we will be deploying an https://github.com/odo-devfiles/springboot-ex[example Spring Boot® component] that uses https://maven.apache.org/install.html[Maven] and Java 8 JDK.
+In this example we will be deploying a sample project associated with the java-spring-boot devfile that uses https://maven.apache.org/install.html[Maven] and Java 8 JDK.
 
-. Download the example Spring Boot® component
+. Create a directory to store the source code for the component:
 +
 [source,sh]
 ----
- $ git clone https://github.com/odo-devfiles/springboot-ex
+ $ mkdir <directory-name>
```

Comment:
Do you mind updating the title / format of "creating the directory" similar to this page: https://docs.openshift.com/container-platform/4.3/cli_reference/openshift_developer_cli/creating-a-single-component-application-with-odo.html

This will help keep all the documentation concise between all tutorial pages.

Reply:
Since I took out `--downloadSource`, we're no longer creating a directory